src/main/java/com/sonyericsson/jenkins/plugins/bfa/model/FailureCause.java

```diff
@@ -86,13 +87,26 @@
      * @param id            the id.
      * @param name          the name of this FailureCause.
      * @param description   the description of this FailureCause.
+     * @param indications   the list of indications
+     */
+    @DataBoundConstructor
+    public FailureCause(String id, String name, String description, List<Indication> indications) {
+        this(id, name, description, null, null, (List<String>)null,
+                indications, null);
+    }
+
+    /**
+     * Standard constructor.
+     *
+     * @param id            the id.
+     * @param name          the name of this FailureCause.
+     * @param description   the description of this FailureCause.
      * @param comment       the comment of this FailureCause.
      * @param lastOccurred  the time at which this FailureCause last occurred.
      * @param categories    the categories of this FailureCause.
      * @param indications   the list of indications
      * @param modifications the modification history of this FailureCause.
      */
-    @DataBoundConstructor
     public FailureCause(String id, String name, String description, String comment,
```

Comment:
```suggestion
    @Deprecated
    public FailureCause(String id, String name, String description, String comment,
```

Reply:
The deprecation in this scenario would only be a flag to anyone coding against it that it is not the one actually used in runtime. It can still be used in tests, just annotated with something if sportbugs or checkstyle complains.